### DISCUSSION TITLE
Presale questions

### DISCUSSION THREAD

**Discussion by hobbycharken** [photoprism/photoprism#4153] [General]
I have the free version up and running without any problem and think of the plus version but have a couple of questions before.
I an a little confused when reading the documentation.
1. Does all users have access to all images or do they have their own library.? 
2. If so, is any change planned so each user has its own space

**Comment by graciousgrey** [comment]
Users of a PhotoPrism instance work on the same library. 
The account roles are intended for situations where you want other people to have access to your library, such as giving family members access to your pictures without granting write permissions or exposing private content.

If you want each user to have his own separate library, you currently would need to set up additional PhotoPrism instances. 

A more convenient library manager is planned for the future:
https://docs.photoprism.app/user-guide/users/libraries/

**Comment by hobbycharken** [comment]
Ok, thanks. That was what I was afraid of. We are agroup of people who want to not using Google Photo for many reasons. We have tested a couple of different applications. Nextcloud and immich. They have no problems with each user have a private library, but lack a lot of other interesting features that Photoprism has. But setting up 15 docker projects is ofc no solution either. Ok, thanks for your answerSkickat från min Galaxy
-------- Originalmeddelande --------Från: Theresa Gresch ***@***.***> Datum: 2024-04-05  14:36  (GMT+02:00) Till: photoprism/photoprism ***@***.***> Kopia: hobbycharken ***@***.***>, Author ***@***.***> Ämne: Re: [photoprism/photoprism] Presale questions (Discussion #4153) 
Users of a PhotoPrism instance work on the same library.
The account roles are intended for situations where you want other people to have access to your library, such as giving family members access to your pictures without granting write permissions or exposing private content.
If you want each user to have his own separate library, you currently would need to set up additional PhotoPrism instances.
A more convenient library manager is planned for the future:
https://docs.photoprism.app/user-guide/users/libraries/
